src/component-helpers.js

```diff
@@ -28,8 +32,13 @@ export function initialize(component) {
     throw new Error('Etch components must implement `update(props, children)`.')
   }
 
+  let virtualElement = component.render()
+  if (!validVirtualElement(virtualElement)) {
+    let namePart = component.constructor && component.constructor.name ? ' in ' + component.constructor.name : ''
+    throw new Error('invalid falsy value ' + virtualElement + ' returned from render()' + namePart)
```

Comment:
Anything that isn't actually a virtual node should probably throw, not just false stuff. Thoughts?